<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(145, 190, 170), stop:1 rgb(161, 255, 191));
}</string>
   </property>
   <widget class="QPushButton" name="daftardatabayi">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>280</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:rgb(0, 197, 145);
color: white;
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Daftar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nama_balita">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Nama Balita</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Jenis Kelamin</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>456</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>NIK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nik_balita">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,7);
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tinggi_badan">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Tinggi Badan (cm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Umur Balita</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="alamat">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,7);
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Berat Badan (kg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>840</width>
      <height>710</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:1px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>840</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>770</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Data Balita</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="umur_balita">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>280</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,7);
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
    <item>
     <property name="text">
      <string>1 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12 Bulan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Tahun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Tahun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Tahun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Tahun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="jenis_kelamin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>280</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
    <property name="currentText">
     <string>Laki-laki</string>
    </property>
    <item>
     <property name="text">
      <string>Laki-laki</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Perempuan</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="tanggal_periksa">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>470</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>450</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Tanggal Periksa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Tanggal Lahir</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="tanggal_lahir">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,7);
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black


</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="date">
     <date>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>daftardatabayi</zorder>
   <zorder>nama_balita</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>error</zorder>
   <zorder>label_5</zorder>
   <zorder>nik_balita</zorder>
   <zorder>tinggi_badan</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>alamat</zorder>
   <zorder>label_8</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>umur_balita</zorder>
   <zorder>jenis_kelamin</zorder>
   <zorder>tanggal_periksa</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>tanggal_lahir</zorder>
  </widget>
 </widget>
 <resources>
  <include location="placeholder.qrc"/>
 </resources>
 <connections/>
</ui>
